src/main/java/school/hei/asa/model/InvoiceDetails.java

```diff
@@ -0,0 +1,5 @@
+package school.hei.asa.model;
+
+import java.time.YearMonth;
+
+public record InvoiceDetails(String id, YearMonth yearMonth, String reference, Worker worker) {}
```

Comment:
I cannot understand what is put inside InvoiceDetails and what is put inside InvoiceForm just by reading their name